src/DEA/BRENDA_mapping.m

```diff
@@ -0,0 +1,206 @@
+%% averaging repeated km values
```

Comment:
Overall, the code is very clean, and the comments do help follow along the logic! Also, the actions performed were done in an acceptable amount of steps. I can see places where you could optimize your code even further (ie reading in the excel file as a cell and doing your analysis) but that's for later.

I would only comment that there are duplicate code structures -> for sabiork / brenda, you use similar code to parse through and clean up the data, which is what I would do as well. As the end product, if it's possible, you would want to figure out a way to make it into a function where it could accept any param from sabiork (pI, Vmax, kcat/Km). Also, as you mentioned earlier, these values can have different units. You can write something up that automatically scales to a fixed unit, then take a mean.

Reply:
Also I just realized this is all in `DEA` (differential expression analysis), so we'll need to find a good home for these files and figure out a good structure for this project. But we'll keep these in here for now.

Reply:
Also great job with the tutorials, and getting into the weeds with how the code works at specific code blocks! 

One minor question for the mapping example - does it let you run the `mapper` function, even if the file name is different? I remember `.m` functions require you to have them be the same name to be functional. If it's not callable as a function, you can remove the function part and add filler arguments for the example ie `modelFilePath = 'INSERT FILE PATH HERE'`. Otherwise, it is good as is.